packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/index.tsx

```diff
@@ -16,7 +17,10 @@ import Success from './template.success'
 
 class Checkout extends PureComponent<Props> {
   componentDidMount () {
-    const amount = this.props.formValues?.amount
+    const dataGoal = find(propEq('name', 'simpleBuy'), this.props.goals)
+    const goalAmount = pathOr('', ['data', 'amount'], dataGoal)
```

Comment:
Would prefer to not use ramda, is it because goals is not type safe?

Reply:
yea, the data is type `any`